src/Data/Vec/Properties.agda

```diff
@@ -155,6 +156,41 @@ take-drop-id (suc m) (x ∷ xs) = begin
     x ∷ xs
   ∎
 
+--------------------------------------------------------------------------------
+-- truncate
+
+truncate-refl : ∀ {n} → (xs : Vec A n) → truncate ≤-refl xs ≡ xs
```

Comment:
Sequences of named arguments don't require an arrow between them, e.g. `∀ {n} → (xs : Vec A n)` becomes `∀ {n} (xs : Vec A n)`. By convention the library doesn't put them in. Any chance you could go through and alter the ones in this file?